### PR TITLE
Replaces all BYOND text macros with spans

### PR DESCRIPTION
**Pull request by Markolie**
This was done using the following Regex pattern:

```
Find: \"\\blue ?([^"]*)\"
Replace: "<span class='notice'>\1</span>"
```

Afterwards I had to push some manual fixes due to in-line string changes.

This also adds the color macros to Travis to make sure these aren't added again.

I still need to do a full review of the files (as this was done almost entirely automatically) to make sure this doesn't cause any issues or has any unclosed tags.

### CODE REVIEW

**Comment by Crazylemon64** [comment]
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa

File: .travis.yml [hunk 1]
```diff
@@ -29,6 +29,7 @@ before_script:
   - chmod +x ./install-byond.sh
   - ./install-byond.sh
 script:
+ - (num=`grep -E '\\\\(red|blue|green|black|b|i[^mc])' **/*.dm | wc -l`; echo "$num escapes (expecting ${MACRO_COUNT} or less)"; [ $num -le ${MACRO_COUNT} ])
```
Comment (Markolie) [inline .travis.yml]: I blatantly ripped this from Bay. Could someone that's more familiar with Travis/grep ensure this works?

**Comment by Markolie** [comment]
Okay, rebased this thing because it was becoming a huge mess. Everything looks good with regards to the macros, but I can't get the Travis check to compile properly. Would appreciate if someone could have a look at that.

**Comment by Markolie** [comment]
Perhaps https://github.com/ParadiseSS13/Paradise/blob/master/_maps/test_away_missions.dm#L9 is triggering the \b[lack] check?

**Comment by Markolie** [comment]
Looks like it is.

**Comment by Markolie** [comment]
Okay, Travis fixed. It now allows for two text macros in the code for the blackmarketpackers file (which triggers the \black macro).

**Comment by KasparoVy** [comment]
:eye: :eye: wild

**Comment by Markolie** [comment]
This is good to go.

**Comment by Markolie** [comment]
No need to review this entire thing by the way if that's what's holding this back. This can't really break anything.

**Comment by tigercat2000** [comment]
It's more about "this will cause conflicts with pretty much every PR and it's a very low priority". Macros don't affect anything but looking bad in the code.

**Comment by Markolie** [comment]
The conflicts are a piece of cake to fix though with Github's new merge conflict resolver. The problematic merge conflicts are icons/maps, which this doesn't touch.

**Comment by ZomgPonies** [comment]
I'd say redo this at the end of the next feature freeze, just before we re-open ourselves to new PRs.

**Comment by Markolie** [comment]
Don't think that's necessary - this PR won't cause many conflicts. If it does, they're ridiculously easy to fix. I'd appreciate this getting merged sometime soon.

**Comment by Fox-McCloud** [comment]
go ahead and fix merge conflicts with this one more time and I"ll merge this.

We can address individual span problems as they crop up.

**Comment by Fox-McCloud** [comment]
Time to conflict with everything.

**Comment by Fox-McCloud** [comment]
Yo @Markolie 

PM me on Discord.

**Pull request merged**